extensions/wikidata/tests/src/org/openrefine/wikidata/qa/scrutinizers/ScrutinizerTest.java

```diff
@@ -79,4 +92,34 @@ public void assertNoWarningRaised() {
     public Set<String> getWarningTypes() {
         return store.getWarnings().stream().map(w -> w.getType()).collect(Collectors.toSet());
     }
+
+    public void setFetcher(ConstraintFetcher fetcher) {
+        scrutinizer.setFetcher(fetcher);
+    }
+
+    public Stream<Statement> conflictsWithParamaterStatementStream(EntityIdValue entityIdValue, PropertyIdValue property, Value propertyValue, PropertyIdValue item, Value itemValue) {
```

Comment:
This method seems to be more generic than its name implies: it can generate any constraint definition which takes two different qualifiers as parameters.

Also I would make the type of the first argument more specific: because it is a value for the P2302 property, it should always be an ItemIdValue.